src/python/pants/backend/docker/target_types.py

```diff
@@ -46,6 +47,24 @@ class DockerBuildArgsField(StringSequenceField):
     )
 
 
+class DockerImageContextRootField(StringField):
+    alias = "context_root"
+    help = (
+        "Specify which directory to use as the Docker build context root, relative to the project "
```

Comment:
"project build root" is a confusing way to put it, since there is no "project" concept in Pants. I'd say just "build root"?